examples/employee-app/src/main/java/com/example/helidon/employee/LoadDbCreds.java

```diff
@@ -0,0 +1,27 @@
+/* Copyright 2019 Oracle and/or its affiliates. All rights reserved. */
+package com.example.helidon.employee;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Properties;
+
+public final class LoadDbCreds {
+    private void LoadsDbCreds(){ }
+
+    public static Properties getProperties(){

```

Comment:
Consider using the Config API instead.
Move the content of `DbCreds.properties` into `application.yaml`, and pass the Config instance to the DBConnection class and EmployeeService.

Reply:
Same as the previous comment. We could do this but it will require quite a bit of rework to our tutorials. So how important is this change to you?

Reply:
The Helidon config API is made for this use-case ; it does not make sense for an Helidon example not to use the config API here.

Reply:
Agreed. We will fix.